modules/git/functions/git-hub-browse

```diff
@@ -42,7 +44,8 @@ if [[ "$reference" == 'HEAD' ]]; then
   reference="$(command git rev-parse HEAD 2> /dev/null)"
 fi
 
-file="$3"
+file="${3:-$(pwd)}"
+file="$(command realpath "$file" | sed "s|^$root||")"
```

Comment:
I'd love suggestions on the 'proper' way to do this in zsh as I'm not experienced with it and shellcheck doesn't zsh yet.

Reply:
`file="${${file:P}#$root}"` will do the work